src/components/ShowCourse.css

```diff
@@ -1,7 +1,10 @@
 .ShowCourse {
-  background-color: #eee;
   margin: 0.5em;
   padding: 0.5em;
   align-self: stretch;
   flex: 1;
-}
\ No newline at end of file
+  background: #eeeeee;
```

Comment:
It would be good to pull these colors out and define some custom properties. I might make this another issue. Here's an example:

```css
:root {
  --bg-color: #eeeeee;
  ...
}

.ShowCourse {
  ...
  background: var(--bg-color);
}